spec/implementation/SerializationP2.cry

```diff
@@ -19,7 +19,7 @@ uncompress_E2_x s_string = x where
     // Step 6 - but not clearly described in the IETF specification
     // AND - some uncertainty about the order of the parts
     // x = reverse (split`{parts=2} (join s_string'))
-    // NOTE: this order, used in ZCash and the code, disagrees with the spec.
+    // NOTE: this order, used in ZCash and the code, disagrees with the spec. TODO: is this comment still accurate?
```

Comment:
I think it is.  It certainly holds for the version of the specs we used, and the latest draft-irtf-cfrg-pairing-friendly-curves has not changed this part.  But we might say the spec is ambiguous, as it refers to both ZCash and the pairing_friendly_curves draft.